weblate/vcs/git.py

```diff
@@ -279,6 +280,20 @@ def configure_remote(self, pull_url, push_url, branch):
         )
         self.branch = branch
 
+    def api_url(self):
+        slug = self.component.repo.split("/")[-1].replace(".git", "")
+        owner = self.component.repo.split("/")[-2]
+        return "https://api.github.com/repos/{0}/{1}".format(owner, slug)
+
+    def configure_fork_remote(self, pull_url, remote_name):
+        """Configure fork remote repository."""
+        self.config_update(
+            # Pull url
+            ('remote "{}"'.format(remote_name), "url", pull_url),
```

Comment:
The pull URL should be still from the original upstream, you only should chnnge push URL here.

Reply:
Okay.